docs/deprecations.md

```diff
@@ -19,11 +19,21 @@ being deprecated.
 | [Several fields of Task.Step are deprecated](https://github.com/tektoncd/pipeline/issues/4737)                                                                                                                               | v0.36.0                                                              | Beta                                                                                                   | Feb 25, 2023                        |
 | [`v1alpha1.Run` is deprecated](https://github.com/tektoncd/community/blob/main/teps/0114-custom-tasks-beta.md)                                                                                                               | v0.43.0                                                              | Alpha                                                                                                  | April 10, 2023 or v0.47.0           |
 | [ClusterTask is deprecated](https://github.com/tektoncd/pipeline/issues/4476)                                                                                                                                                | v0.41.0                                                              | Beta                                                                                                   | July 13, 2023                       |
 | [`pipelineRef.bundle` and `taskRef.bundle` are deprecated](https://github.com/tektoncd/pipeline/issues/5514)                                                                                                                 | v0.41.0                                                              | Alpha                                                                                                  | July 13, 2023                       |
-| [`taskrun.status.cloudEvents` are deprecated](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)                                                                                     | v0.44.0                                                              | Alpha                                                                                                  | Oct 11, 2023                        |
+
+### Deprecate PipelineResources related features:
+
+The following features are deprecated as part of the deprecation of PipelineResources.
+See [TEP-0074](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md) for more information.
+
+- the fields`task.spec.resources`, `taskRun.spec.resources`, `pipeline.spec.resources`, `pipelineRun.spec.resources`, and `taskRun.status.cloudEvents`
+
+- The "kubeconfigwriter" and "imagedigestexporter" images, used by the Cluster PipelineResource and Image PipelineResource, respectively
```

Comment:
suggest adding links to [imagedigestexporter](https://github.com/tektoncd/pipeline/blob/release-v0.43.x/pkg/apis/pipeline/images.go#L46) and [KubeconfigWriterImage](https://github.com/tektoncd/pipeline/blob/release-v0.43.x/pkg/apis/pipeline/images.go#L36)